app/models/external_registries/stop_heling_result.rb

```diff
@@ -12,21 +12,15 @@ class StopHelingResult
       :location,
       :object,
       :registry_id,
-      :registry_name,
-      :registry_url,
       :source_name,
-      :source_unique_id,
-      :url
+      :source_unique_id
```

Comment:
There is no need for this attribute.

This should be `external_id`. The attribute that is being put into `external_id` in the PR right now should actually be stored in `additional_registration`

Reply:
Please elaborate

Reply:
The original intention of the attributes was that they would do for the purposes specified - where `external_id` is the unique source id of the registration.

The additional registration number, if we do need to store that, would be the displayed id if they have it.

Reply:
> external_id is the unique source id of the registration.

If I understand correctly, the `external_id` column should store an `external_registry_bike` record's uniquely identifying id in the associated bike registry (in this case stopheling.nl) — is that correct? 

That's what's implemented currently. (Hence the uniqueness validation for `external_id` scoped to `registry`)

`:source_unique_id` and `:source_name` are specific to stopheling.nl and refer to the source from the perspective of stopheling.nl (there are multiple sources for records within stopheling). I would think that one or both of those would be stored as `additional_registration` if we want to store them at all. Please advise if that's mistaken.

```
        "BronNaam" => "POLITIE BRABANT-NOORD",
        "BronUniekID" => "458171",
```

```
        "BronNaam" => "POLITIE KENNEMERLAND",
        "BronUniekID" => "83760",
```

Reply:
Ok. Sorry! I miss-understood.